addons/mail/static/src/core/common/thread.xml

```diff
@@ -36,7 +36,7 @@
                         thread="props.thread"
                         messageEdition="props.messageEdition"
                         isFirstMessage="msg_first"
-                        hasActions="props.messageActions"
+                        hasActions="props.messageActions and !msg.eq(props.thread.initial_message_id)"
```

Comment:
Why not allow actions on it? I would expect to sync with original message in all contexts.

For example adding a reaction there or in main thread should work and sync. I could also reply in subthread to the (original / from) message of subthread. We could even edit or delete it (hopefully the delete of [initial/from] message of subthread is robust? I haven't checked yet)

Reply:
It would indeed work. I agree that most actions makes sense, I found that replying was a bit weird as you would compose the message in the sub-thread but post it on the main one. What do you think?

Reply:
Ok to not allow message actions for the initial / from message of subthread :)